tracker-radar-tests/TR-domain-matching/domain_matching_tests.json

```diff
@@ -0,0 +1,163 @@
+            },
+            {
+                "name": "tracker loads ignore",
+                "siteURL": "https://bad.third-party.site/",
+                "requestURL": "https://ignore.test/",
+                "requestType": "script",
+                "expectAction": "ignore"
+            },
+            {
+                "name": "ignore image exception loads tracker image",
+                "siteURL": "https://ignore.test/",
+                "requestURL": "https://bad.third-party.site/",
+                "requestType": "image",
+                "expectAction": "ignore"
+            },
+            {
+                "name": "ignore image exception doesn't load tracker script",
+                "siteURL": "https://ignore.test/",
+                "requestURL": "https://bad.third-party.site/",
+                "requestType": "script",
+                "expectAction": "block"
+            },
+            {
+                "name": "notignore doesn't load tracker",
+                "siteURL": "https://notignore.test/",
+                "requestURL": "https://bad.third-party.site/",
+                "requestType": "script",
+                "expectAction": "block"
+            },
+            {
+                "name": "random blocks tracker",
+                "siteURL": "https://randomsite123.com/",
+                "requestURL": "https://bad.third-party.site/",
+                "requestType": "script",
+                "expectAction": "block"
+            },
+            {
+                "name": "sub.ignore loads tracker",
+                "siteURL": "https://sub.ignore.test/",
+                "requestURL": "https://bad.third-party.site/",
+                "requestType": "image",
+                "expectAction": "ignore"
+            },
+            {
+                "name": "tracker loads ignore resource type",
+                "siteURL": "https://bad.third-party.site/",
+                "requestURL": "https://ignore.test/",
+                "requestType": "image",
+                "expectAction": "ignore"
+            },
+            {
+                "name": "tracker blocks non-ignore resource type",
+                "siteURL": "https://bad.third-party.site/",
+                "requestURL": "https://ignore.test/",
+                "requestType": "script",
+                "expectAction": "ignore"
+            },
+            {
+                "name": "tracker loads random",
+                "siteURL": "https://bad.third-party.site/",
+                "requestURL": "https://random.test/",
+                "requestType": "script",
+                "expectAction": null
+            },
+            {
+                "name": "random does not load tracker",
+                "siteURL": "https://random.test/",
+                "requestURL": "https://bad.third-party.site/",
+                "requestType": "script",
+                "expectAction": "block"
+            },
+            {
+                "name": "random loads itself",
```

Comment:
This is testing a same party request to an unknown domain (non-tracker) where 'same party ignore' is testing a same party request for a known tracker?
 https://github.com/duckduckgo/privacy-reference-tests/pull/1/files#diff-1c95c4a4033bb55c3e220061a1f70aba4195c9624933a9389f88732cdd35809bR31

Reply:
Correct, its a pretty dumb test but it does help protect against a regression in the default behavior